frontend/packages/dev-console/src/components/monitoring/alerts/SilenceDurationDropdown.tsx

```diff
@@ -35,6 +35,8 @@ const durations = {
   '1d': '1 day',
 };
 
+const isInternalLabel = ({ name }: { name: string }) => name !== 'prometheus';
```

Comment:
I think `prometheus` is actually an external label, not an internal label (see https://prometheus.io/docs/prometheus/latest/configuration/configuration/)

Reply:
yes indeed, maybe is the function name that made it confusing as is an excluding filter.